<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VentanaPrincipal</class>
 <widget class="QMainWindow" name="VentanaPrincipal">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>820</width>
    <height>540</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>820</width>
    <height>540</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>820</width>
    <height>540</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>.AppleSystemUIFont</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>MicroStream</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">QWidget#centralwidget{
background-image: url(:/resources/Imagenes/fondoMain.png);
}

QPushButton{
	background-color: rgb(179, 179, 179);
border-radius:15px;
border:1px solid;
border-color: rgb(102, 102, 102);
color:#ffffff;
font-size:13px;
}

QPushButton:hover{
background-color: rgba(16, 128, 128, 240);
}</string>
   </property>
   <widget class="QPushButton" name="botonAjustes">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>310</y>
      <width>80</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Ajustes</string>
    </property>
    <property name="autoDefault">
     <bool>false</bool>
    </property>
    <property name="default">
     <bool>false</bool>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="botonIniciar">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>370</y>
      <width>80</width>
      <height>31</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="text">
     <string>GO!</string>
    </property>
    <property name="autoDefault">
     <bool>false</bool>
    </property>
    <property name="default">
     <bool>false</bool>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QComboBox" name="selectorPelicula">
    <property name="geometry">
     <rect>
      <x>302</x>
      <y>260</y>
      <width>294</width>
      <height>24</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>1</height>
     </size>
    </property>
    <property name="editable">
     <bool>false</bool>
    </property>
    <item>
     <property name="text">
      <string>Dragon Adventure</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Big Buck Bunny</string>
     </property>
    </item>
   </widget>
   <widget class="QRadioButton" name="elegirVideos">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>260</y>
      <width>100</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QRadioButton" name="elegirUrl">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>310</y>
      <width>100</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="campoUrl">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>310</y>
      <width>281</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Introduce la URL de un fichero .mpd</string>
    </property>
   </widget>
   <widget class="QLabel" name="mensaje">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>440</y>
      <width>241</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:14pt; color:#fc0107;&quot;&gt;Por favor, selecciona una opcion.&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
